src/components/navigation/Routes/Routes.tsx

```diff
@@ -2,6 +2,9 @@ import React from 'react'
 import { Redirect, Route, RouteComponentProps, Switch, withRouter } from 'react-router-dom'
 import styled from 'styled-components'
 
+//import ScrollArea from 'react-scrollbar'
```

Comment:
@maria-vslvn I also disabled the scroll container here as it was causing some components and hooks to re-render and there were http requests firing all the time. We should debug this issue.